TWLight/resources/templates/resources/partner_users.html

```diff
@@ -0,0 +1,86 @@
+{% extends "base.html" %}
+{% load i18n %}
+
+{% block content %}
+  {% comment %} Translators: This is the heading of a page listing editors who have applications for this partner ({{ object }}). {% endcomment %}
+  <h1>
+  {% blocktrans with partner=object.company_name %}
+  {{ object }} approved users
+  {% endblocktrans %}
+  </h1>
+
+  <div class="alert alert-info">
+    {% comment %} Translators: This message is shown on pages where coordinators can see personal information about applicants. {% endcomment %}
+    {% trans 'Coordinators: This page may contain personal information such as real names and email addresses. Please remember that this information is confidential.' %}
+  </div>
+
+  {% if approved_applications|length > 0 %}
+    {% comment %} Translators: On the page listing approved users for a partner, this is the title of the section listing applications with the 'approved' status {% endcomment %}
+    <h2>{% trans "Approved applications" %}</h2>
+    <table class="table table-sm">
+      <thead>
+        <tr>
+          <th>User</th>
+          <th>Email</th>
+          <th>Application date</th>
+          <th>Application approved</th>
+          <th>Renewal?</th>
+          {% if partner_streams %}
+            <th>Stream</th>
+          {% endif %}
+        </tr>
+      </thead>
+      <tbody>
+      {% for application in approved_applications %}
+        <tr>
+          <td><a href="https://meta.wikimedia.org/wiki/User:{{ application.editor.wp_username }}">
+            {{ application.editor.wp_username }}
+          </a></td>
+          <td>{{ application.editor.user.email }}</td>
+          <td><a href="{% url 'applications:evaluate' pk=application.pk %}">
+            {% if application.imported %}<i>Imported</i>{% else %}{{ application.date_created }}{% endif %}
+          </a></td>
+          <td>{{ application.date_closed }}</td>
+          <td>{% if application.parent %}<i>Renewal</i>{% endif %}</td>
+          {% if application.specific_stream %}<td>{{ application.specific_stream }}</td>{% endif %}
+        </tr>      
+      {% endfor %}
+      </tbody>
+    </table>
+  {% endif %}
+
+  {% if sent_applications|length > 0 %}
+    {% comment %} Translators: On the page listing approved users for a partner, this is the title of the section listing applications with the 'sent' status {% endcomment %}
+    <h2>{% trans "Sent applications" %}</h2>
+    <table class="table table-sm">

```

Comment:
the text in this table should be marked for translation. Have you looked at this on a phone? Make use of a responsive grid instead of a table wherever possible.